PHYSOR26/common_mesh.i

```diff
@@ -0,0 +1,126 @@
+ref_mesh_file_name=gt.e-s011
+test_mesh_file_name=hex_same_flux_high_err_ma.e-s010
+
+[Mesh]
+  [generated_mesh]
+     type = GeneratedMeshGenerator
+    nx = 20
+    ny = 4
+    nz = 4
+    xmin = 0.0
+    xmax = 100.0
+    ymin = 0.0
+    ymax = 10.0
+    zmin = 0.0
+    zmax = 10.0
+    dim = 3
+  []
+[]
```

Comment:
Is there a way to ensure this is the same mesh as the original mesh for each of your cases?